plugins/desktop/src/desktop-tracker-query.vala

```diff
@@ -21,7 +29,11 @@ private class Games.DesktopTrackerQuery : Object, TrackerQuery {
 		return true;
 	}
 
-	public Game game_for_cursor (Tracker.Sparql.Cursor cursor) throws Error {
+	public Game game_for_cursor (string uri) throws Error {
```

Comment:
Which ones? We need to remove it from everywhere as we replaced it with `process_cursor()` and `foreach_game()`.